R/kobo_dico.R

```diff
@@ -24,187 +24,187 @@
 #'
 
 kobo_dico <- function(form) {
-
+  
   #kobo_form(formid, user = user, api = api)
   cat("\n Your form should be placed within the `data` folder. \n \n")
   # read the survey tab of ODK from
-  form_tmp <- paste0("data/",form)
-
-
+  form_tmp <- paste0(mainDir,"/data/",form)
```

Comment:
I think you need to define the mainDir variable before using it, please replace this line with the below code
```
mainDir <- gsub("/code/shiny_app", "",  getwd()) 
mainDir <- gsub("/inst/shiny_app", "",  mainDir) 
form_tmp <- paste(mainDir, "data", form, sep = "/", collapse = "/")
```

Reply:
Added